python/llm/example/GPU/PyTorch-Models/Model/qwen2-vl/generate.py

```diff
@@ -0,0 +1,107 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+import time
+import argparse
+
+from transformers import AutoTokenizer
+from ipex_llm import optimize_model
+import numpy as np
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Qwen2-VL-7B-Instruct')
+    parser.add_argument('--repo-id-or-model-path', type=str, default="Qwen/Qwen2-VL-7B-Instruct",
+                        help='The huggingface repo id for the Qwen2-VL model to be downloaded'
+                             ', or the path to the huggingface checkpoint folder')
+    parser.add_argument('--prompt', type=str, default="Describe this image.",
+                        help='Prompt to infer') 
+    parser.add_argument('--n-predict', type=int, default=32,
+                        help='Max tokens to predict')
+
+    args = parser.parse_args()
+    model_path = args.repo_id_or_model_path
+
+
+    from transformers import Qwen2VLForConditionalGeneration, AutoTokenizer, AutoProcessor
+    from qwen_vl_utils import process_vision_info
+    from ipex_llm import optimize_model
+    model = Qwen2VLForConditionalGeneration.from_pretrained(model_path,
+                                                 trust_remote_code=True,
+                                                 torch_dtype = 'auto',
+                                                 use_cache=True)
+    model = optimize_model(model, low_bit='sym_int4')
```

Comment:
Please refer to [here](https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/PyTorch-Models/Model/llama3/generate.py#L57) to add several more augments :)

Reply:
`low_cpu_mem_usage=True` doesn't seem to apply to `Qwen2VLForConditionalGeneration.from_pretrained`:
![image](https://github.com/user-attachments/assets/4fa62bb6-5863-4ed8-b351-990e038f5f93)
error persists after installation of `accelerate`